app/models/concerns/draft_version.rb

```diff
@@ -9,6 +9,61 @@ def self.build_draft_version(attrs = {})
       new(attrs)
     end
 
-  end
+    def self.draft_pid(pid)
```

Comment:
Could you rebase?  This looks like the old code.  These methods have since been moved to app/models/pid_utils.rb.